pkgs/development/python-modules/manga-ocr/default.nix

```diff
@@ -1,48 +1,30 @@
 {
   lib,
   fetchFromGitHub,
-  buildPythonPackage,
-  fire,
-  fugashi,
-  jaconv,
-  loguru,
-  numpy,
-  pillow,
-  pyperclip,
-  torch,
-  transformers,
-  unidic-lite,
-  pythonOlder,
+  python3Packages,
 }:
+
+with python3Packages;
+
 buildPythonPackage rec {
   pname = "manga-ocr";
   version = "0.1.11";
```

Comment:
I think you forgot to cherry-pick the version change here.

Reply:
ah right, nice catch!